xbmc/windowing/XBMC_events.h

```diff
@@ -78,6 +80,10 @@ typedef struct XBMC_MoveEvent {
 	int y;		/* New y position */
 } XBMC_MoveEvent;
 
+struct XBMC_ModeChangeEvent {
```

Comment:
whole file is wrong format then, do you think it makes sense to only change this line?